tests/integration/acp/nac/dac_access_by_empty_node_user_with_nac_off_test.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2025 Democratized Data Foundation
```

Comment:
thought: This, and the others, have very long file names, and this reduces readability.   A lot of the time all people are going to see in their IDEs is `dac_access_by_empty...[hidden]`.  There are also a lot of files in this directory, and I think we would benefit from a few more sub directories, with short names in the future.